Default (Windows).sublime-keymap

```diff
@@ -0,0 +1,3 @@
+[
+	{ "keys": ["ctrl+shift+v"], "command": "paste_from_history_edit" }
```

Comment:
I don't know if we should bind keys by default - what do you think? Maybe we should let people bind their own keys?

Reply:
People can always rebind keys how they like, but I consider that this [paste from history] operation is more needed than paste_and_indent which is assigned by default in SublimeText at ctrl+shift+v.
Besides paste_and_indent is somewhat broken and can not be reliably used, for example:
If you have code like this:
```
if nesting_level == 0:
    break
```
and put cursor after last line with `break`, then command past_and_indent align `new_code` wrong:
```
if nesting_level == 0:
    break
    new_code
```
Another example:
If you have
```
if nesting_level == 0: # comment
    break
```
then cut line with `break`:
```
if nesting_level == 0: # comment
```
and immediately execute past_and_indent command:
```
if nesting_level == 0: # comment
break
```
<br>
So, I would rather go with regular paste command and realign all code manually.